src/print.jl

```diff
@@ -283,39 +283,39 @@ end
 
 Base.show(io::IO, a::GenericAffExpr) = print(io, aff_string(REPLMode,a))
 function Base.show(io::IO, ::MIME"text/latex", a::GenericAffExpr)
-    print(io, wrap_in_math_mode(aff_string(IJuliaMode, a)))
+    print(io, _wrap_in_math_mode(aff_string(IJuliaMode, a)))
 end
 
 function aff_string(mode, a::GenericAffExpr, show_constant=true)
```

Comment:
I'd vote to unexport `aff_string`.

Reply:
Then we might as well remove it, see https://github.com/JuliaOpt/JuMP.jl/pull/1836